en/releases/1.15.md

```diff
@@ -17,34 +17,69 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 
 ## Major Changes
 
-- TBD
+- [New] Throw Mode [PX4-Autopilot#21170](https://github.com/PX4/PX4-Autopilot/pull/21170)
+- Improved Optical Flow Support [PX4-Autopilot#21769](https://github.com/PX4/PX4-Autopilot/pull/21769)
```

Comment:
@mrpollo Content looks great. At the end we should have a consistency update for how we present the autopilot link as many are different.. I prefer:

```suggestion
- [New] Throw Mode ([PX4-Autopilot#21170](https://github.com/PX4/PX4-Autopilot/pull/21170)).
- Improved Optical Flow Support ([PX4-Autopilot#21769](https://github.com/PX4/PX4-Autopilot/pull/21769))
```